### PR TITLE
NET-225 Replace static classes with enum if possible and fix naming

### PR DESCRIPTION
**Pull request by BohdanBuhrii**

### CODE REVIEW

File: TransIT.BLL/Helpers/Roles.cs
Comment (igolovchuk) [inline TransIT.BLL/Helpers/Roles.cs]: - remove 'static'
- rename to 'Role'

File: TransIT.BLL/Helpers/Roles.cs
Comment (igolovchuk) [inline TransIT.BLL/Helpers/Roles.cs]: - rename to 'Schema'

File: TransIT.BLL/Helpers/Configuration/AzureConfig.cs
Comment (igolovchuk) [inline TransIT.BLL/Helpers/Configuration/AzureConfig.cs]: remove 'static', add modifier

File: TransIT.BLL/Helpers/Configuration/AzureConfig.cs
Comment (igolovchuk) [inline TransIT.BLL/Helpers/Configuration/AzureConfig.cs]: -remove the word 'Storage' from these constants

File: TransIT.BLL/Helpers/Warranties.cs
Comment (igolovchuk) [inline TransIT.BLL/Helpers/Warranties.cs]: use boolean if possible, change the DAL entity as well

File: TransIT.BLL/Services/ImplementedServices/IssueService.cs
Comment (igolovchuk) [inline TransIT.BLL/Services/ImplementedServices/IssueService.cs]: try to change type to boolean

**Review by igolovchuk — CHANGES_REQUESTED** [review]
please look at the comments and try to resolve them

File: TransIT.BLL/Helpers/Roles.cs
Comment (BohdanBuhrii) [reply TransIT.BLL/Helpers/Roles.cs]: If rename to "Role" then will be a conflict with Issue.Role, is this ok?

Comment (igolovchuk) [reply TransIT.BLL/Helpers/Roles.cs]: I don't see Issue.Role, where it's located?

Comment (BohdanBuhrii) [reply TransIT.BLL/Helpers/Roles.cs]: Sorry, I was wrong, I meant Entities.Role

Comment (igolovchuk) [reply TransIT.BLL/Helpers/Roles.cs]: Rename RoleConsts to RoleNames

**Review by igolovchuk — CHANGES_REQUESTED** [review]
Rename RoleConsts to RoleNames and revert last changes regarding using Namespaces to fix conflict names issue

File: TransIT.BLL/Helpers/Warranties.cs
Comment (igolovchuk) [inline TransIT.BLL/Helpers/Warranties.cs]: - remove this class and in usages just set 'true' or 'false', eg. issue.Warranty = true;

File: TransIT.BLL/Services/ImplementedServices/IssueService.cs
Comment (igolovchuk) [inline TransIT.BLL/Services/ImplementedServices/IssueService.cs]: issue.Warranty = true;

**Review by igolovchuk — CHANGES_REQUESTED** [review]
- see comments

**Review by igolovchuk — DISMISSED** [review]

**Review by VitalikRibiy — APPROVED** [review]

**Review by igolovchuk — APPROVED** [review]

**Review by Vladacdc — APPROVED** [review]

**Comment by Vladacdc** [comment]
DO NOT MERGE UNTIL CLIENT IS READY FOR THIS MERGE

**Pull request merged**
